prac_05/wimbledon.py

```diff
@@ -0,0 +1,42 @@
+FILENAME = "wimbledon.csv"
+
+
+def main():
+    records = get_records(FILENAME)
+    champion_to_count, countries = process_records(records)
+    display_results(champion_to_count, countries)
+
+
+def process_records(records):
+    champion_to_count = {}
+    countries = set()
```

Comment:
No need to set it to nothing then set it to the values. This can all be done in one line